src/main/java/com/exasol/cloudwatch/CloudWatchAdapter.java

```diff
@@ -0,0 +1,82 @@
+package com.exasol.cloudwatch;
+
+import java.sql.*;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.ScheduledEvent;
+import com.exasol.errorreporting.ExaError;
+
+/**
+ * This class is the entry point for the Lambda function of the cloud watch adapter.
+ */
+public class CloudWatchAdapter implements RequestHandler<ScheduledEvent, Void> {
+
+    private static final Logger logger = LoggerFactory.getLogger(CloudWatchAdapter.class);
+    private final String exasolStatisticsSchemaOverride;
+    private final AdapterConfiguration configuration;
+
+    /**
+     * Create a new instance of {@link CloudWatchAdapter}.
+     */
+    public CloudWatchAdapter() {
+        this(null);
+    }
+
+    /**
+     * Constructor for testing.
+     * 
+     * @param exasolStatisticsSchemaOverride if null EXA_STATISITCS is used. This parameter allows you to test this
+     *                                       connector with a * predefined SCHEMA instead of the unmodifiable live
+     *                                       statistics.
+     */
+    CloudWatchAdapter(final String exasolStatisticsSchemaOverride) {
+        this.configuration = new AdapterConfiguration();
+        this.exasolStatisticsSchemaOverride = exasolStatisticsSchemaOverride;
+
```

Comment:
```suggestion
```

Reply:
If this is only required in the testing, can you add setter for the override?

Reply:
yes, but I wanted to keep everything final.